op-deployer/pkg/deployer/pipeline/l2genesis.go

```diff
@@ -22,6 +22,9 @@ import (
 )
 
 type l2GenesisOverrides struct {
+	UseCustomGasToken                        bool                      `json:"useCustomGasToken"`
+	GasPayingTokenName                       string                    `json:"gasPayingTokenName"`
+	GasPayingTokenSymbol                     string                    `json:"gasPayingTokenSymbol"`
```

Comment:
Not sure what the convention is here, but I would think it a good idea to document which set of configuration options pertain to CGT, for future reference.


```suggestion
type l2GenesisOverrides struct {
    // CGT
	UseCustomGasToken                        bool                      `json:"useCustomGasToken"`
	GasPayingTokenName                       string                    `json:"gasPayingTokenName"`
	GasPayingTokenSymbol                     string                    `json:"gasPayingTokenSymbol"`
```

Reply:
Fixed https://github.com/ethereum-optimism/optimism/pull/17412/commits/dfe5282a958e2aeac3fce599e4a2f0cb174c6cb5